plugin.audio.soundcloud/resources/settings.xml

```diff
@@ -0,0 +1,88 @@
+                </setting>
+            </group>
+            <group id="2" label="30011">
+                <setting id="search.items.size" type="string" label="30012" help="">
+                    <level>0</level>
+                    <default>20</default>
+                    <constraints>
+                        <options>
+                            <option>5</option>
+                            <option>10</option>
+                            <option>15</option>
+                            <option>20</option>
+                            <option>25</option>
+                            <option>30</option>
+                            <option>40</option>
+                            <option>50</option>
+                        </options>
+                    </constraints>
+                    <control type="spinner" format="string"/>
+                </setting>
+                <setting id="search.history.size" type="string" label="30013" help="">
+                    <level>0</level>
+                    <default>10</default>
+                    <constraints>
+                        <options>
+                            <option>0</option>
+                            <option>5</option>
+                            <option>10</option>
+                            <option>15</option>
+                            <option>20</option>
+                            <option>30</option>
+                            <option>50</option>
+                        </options>
+                    </constraints>
+                    <control type="spinner" format="string"/>
+                </setting>
+            </group>
+            <group id="3" label="30060">
+                <setting id="apiv2.client_id" type="string" label="30061" help="">
+                    <level>3</level>
+                    <default/>
+                    <constraints>
+                        <allowempty>true</allowempty>
+                    </constraints>
+                    <control type="edit" format="string">
+                        <heading>30061</heading>
+                    </control>
+                </setting>
+                <setting id="apiv2.locale" type="integer" label="30062" help="">
+                    <level>2</level>
+                    <default>0</default>
+                    <constraints>
+                        <options>
+                            <option label="30063">0</option>
+                            <option label="30064">1</option>
+                        </options>
+                    </constraints>
+                    <control type="spinner" format="string"/>
+                </setting>
+            </group>
+            <group id="4" label="30070">
+                <setting id="cache.clear" type="action" label="30071" help="">
+                    <level>1</level>
+                    <data>RunPlugin(plugin://plugin.audio.soundcloud/settings/cache/clear/)</data>
```

Comment:
@ronie I noticed I can't use `$ID` anymore. Is this a bug or was that a deliberate decision?

See https://github.com/jaylinski/kodi-addon-soundcloud/blob/v3.2.0/resources/settings.xml#L15.

Should probably be mentioned in https://kodi.wiki/view/Add-on_settings_conversion.

Reply:
i don't know i'm afraid... the wiki page for the 'old' addon settings does not mention $ID either.
maybe @Montellese can tell.

Reply:
@ronie I opened a PR regarding the `$ID` replacement: https://github.com/xbmc/xbmc/pull/18959

This addon-PR can be merged in the meantime, if you approve it. :)